### ISSUE TITLE
[question] Do I need to run this on every host?

### ISSUE THREAD

**Issue by kevin-coyle** [adi90x/rancher-active-proxy#46]
I want to have multiple hosts on my infrastructure. Should I put RAP on each host?

**Comment by adi90x** [comment]
Yes, you can put rancher on all host you can then load balance traffic from dns to the different host IP.
Or you can run one RAP on main host , send all traffic to this host , and let it forward request to containers/services spawn on slave host
